app/services/experimentation/runner.rb

```diff
@@ -1,26 +1,38 @@
-class ExperimentControlService
-  LAST_EXPERIMENT_BUFFER = 14.days.freeze
-  PATIENTS_PER_DAY = 10_000
-  BATCH_SIZE = 100
-
-  class << self
-    def start_current_patient_experiment(name:, percentage_of_patients: 100)
-      experiment = Experimentation::Experiment.find_by(name: name, experiment_type: "current_patients", state: [:new, :running])
-      unless experiment
-        Sentry.capture_message("Experiment #{name} not found and may need to be removed from scheduler.")
+# require_relative "stale_patient_selection"
+
+module Experimentation
+  class Runner
```

Comment:
Moving this into Experimentation makes a bunch of other things more clear and easier to follow below...